yeelight/flow.py

```diff
@@ -44,6 +44,10 @@ def __init__(self, count=0, action=Action.recover, transitions=None):
         self.action = action
         self.transitions = transitions
 
+        # Note, main depends on us, so we cannot import BulbException here.
+        if len(self.transitions) > 10:
+            raise Exception("More transactions (%s) than maximum of 10." % len(self.transitions))
```

Comment:
Hmm. That's a possibility indeed. Well, in that case maybe logging.error() to make it visibile for people wondering why their hacks are broken? Just having it in the docs alone, esp. while doing some dev work isn't that helpful.